tests/tabescape_dflt.sh

```diff
@@ -1,18 +1,29 @@
 #!/bin/bash
-echo ===============================================================================
-echo \[tabescape_dflt.sh\]: test for default tab escaping
+# add 2018-06-29 by Pascal Withopf, released under ASL 2.0
 . $srcdir/diag.sh init
-. $srcdir/diag.sh generate-HOSTNAME
+. $srcdir/diag.sh generate-conf
+. $srcdir/diag.sh add-conf '
+module(load="../plugins/imtcp/.libs/imtcp")
+input(type="imtcp" port="13514" ruleset="ruleset1")
 
-./nettester -ttabescape_dflt -iudp
-if [ "$?" -ne "0" ]; then
-  echo erorr in udp run
-  exit 1
-fi
+template(name="outfmt" type="string" string="%msg%\n")
 
-echo test via tcp
-./nettester -ttabescape_dflt -itcp
-if [ "$?" -ne "0" ]; then
-  echo erorr in tcp run
-  exit 1
-fi
+ruleset(name="ruleset1") {
+	action(type="omfile" file="rsyslog.out.log"
+	       template="outfmt")
+}
+
+'
+. $srcdir/diag.sh startup
+. $srcdir/diag.sh tcpflood -m1 -M "\"<167>Mar  6 16:57:54 172.20.245.8 test: before HT	after HT (do NOT remove TAB!)\""
```

Comment:
Is there actually a tab character present? Doesn't look so.

Reply:
There is a tab in front of "after". It doesn't look like it but I have confirmed this.